src/main/java/com/avispl/symphony/dal/avdevices/wirelesspresentation/mersive/solsticepodgen3/SolsticePodGen3Communicator.java

```diff
@@ -0,0 +1,1256 @@
+						localCacheMapOfPropertyNameAndValue.put(property.getName(), getMinutes(value));
+						break;
+					default:
+						localCacheMapOfPropertyNameAndValue.put(property.getName(), value);
+				}
+			}
+		} catch (Exception e) {
+			logger.error("Error while retrieve data: " + e);
+			localCacheMapOfPropertyNameAndValue.clear();
+		}
+	}
+
+	/**
+	 * Retrieves the configuration command from the API and stores the JSON response in the 'configResponse' field.
+	 *
+	 * Throws 'ResourceNotReachableException' if login fails and logs other exceptions.
+	 */
+	private void retrieveConfigurationCommand() {
+		try {
+			configResponse = doGet(SolsticeCommand.CONFIG_COMMAND, JsonNode.class);
+		} catch (Exception e) {
+			failedMonitor.put(SolsticeCommand.CONFIG_COMMAND, e.getMessage());
+			logger.error("Error when retrieve configuration command", e);
+		}
+	}
+
+	/**
+	 * Retrieves the statistics command from the API and stores the JSON response in the 'statisticResponse' field.
+	 *
+	 * Throws 'ResourceNotReachableException' if login fails and logs other exceptions.
+	 */
+	private void retrieveStatisticsCommand() {
+		try {
+			String request = SolsticeCommand.STATS_COMMAND + (StringUtils.isNotNullOrEmpty(this.getPassword()) ? "?password=" + this.getPassword() : SolsticeConstant.EMPTY);
+			statisticResponse = doGet(request, JsonNode.class);
+		} catch (FailedLoginException e) {
+			throw new ResourceNotReachableException("Failed to login, please check the password", e);
+		} catch (Exception e) {
+			failedMonitor.put(SolsticeCommand.STATS_COMMAND, e.getMessage());
+			logger.error("Error when retrieve statistics command", e);
+		}
+	}
+
+	/**
+	 * populate monitoring and controlling data
+	 *
+	 * @param stats the stats are list of Statistics
+	 * @param controlStats the control stats are list of Statistics
+	 * @param advancedControllableProperties the advancedControllableProperties are list AdvancedControllableProperty instance
+	 */
+	private void populateMonitoringAndControllingData(Map<String, String> stats, Map<String, String> controlStats, List<AdvancedControllableProperty> advancedControllableProperties) {
+		String value;
+		String propertyName;
+		for (SolsticePropertiesList property : SolsticePropertiesList.values()) {
+			value = localCacheMapOfPropertyNameAndValue.get(property.getName());
+			propertyName = property.getGroup().concat(property.getName());
+			if (StringUtils.isNotNullOrEmpty(value)) {
+				switch (property) {
+					case RESET_KEY:
+						addAdvanceControlProperties(advancedControllableProperties, controlStats, createButton(propertyName, SolsticeConstant.RESTART, SolsticeConstant.RESTARTING, SolsticeConstant.GRACE_PERIOD));
+						break;
+					case SET_DEFAULT_BACKGROUND:
+						addAdvanceControlProperties(advancedControllableProperties, controlStats, createButton(propertyName, SolsticeConstant.SET, SolsticeConstant.SETTING, SolsticeConstant.GRACE_PERIOD));
+						break;
+					case SCHEDULED_DAILY_REBOOT:
```

Comment:
is it ok there's no breaks to the blank cases? It's going to get fallen down to USE_24_HOUR_TIME_FORMAT

Reply:
> is it ok there's no breaks to the blank cases? It's going to get fallen down to USE_24_HOUR_TIME_FORMAT

It will get fallen down to USE_24_HOUR_TIME_FORMAT